proto/imuachain/oracle/v1/prices.proto

```
@@ -1,10 +1,10 @@
 syntax = "proto3";
-package exocore.oracle.v1;
+package imuachain.oracle.v1;
 
-import "exocore/oracle/v1/price.proto";
+import "imuachain/oracle/v1/price.proto";
```

Buf (1.47.2)

import "imuachain/oracle/v1/price.proto": file does not exist

(COMPILE)